src/scss/styles.scss

```diff
@@ -0,0 +1,59 @@
+@import "variables";
+
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
+
+/* Bootstrap Sign In Example */
+
+.form-signin {
```

Comment:
Good job! but then all these styles are just CSS in an SCSS file. Your style would be neater with pure scss like

```
.form-signin {
    max-width: 400px;
    padding: 15px;

    .form-floating:focus-within {
          z-index: 2;
     }
```

Read more at https://noroff-content.gitlab.io/feu/css-frameworks/intro-to-sass.html#nesting

Reply:
Fixed as suggested. Removed classed that wasn't used. Not sure how nest inputs.